iam_ltk.go

```diff
@@ -21,48 +21,48 @@ type GetLongTermKeysResponse struct {
 	LongTermKeys []LongTermKey `json:"longTermKeys"`
 }
 
+// BaseLongTermKeyResponse encapsulates shared response fields
+type BaseLongTermKeyResponse struct {
+	Action              string `json:"action,omitempty"`
+	AddedIAMUserToGroup bool   `json:"addedIAMUserToGroup,omitempty"`
+	PartialError        bool   `json:"partialError,omitempty"`
+}
+
 // CreateLongTermKey represents the response from API
 type CreateLongTermKey struct {
-	Account             string `json:"account"`
-	Action              string `json:"action"`
-	IAMUserName         string `json:"iamUserName"`
-	IAMUserArn          string `json:"iamUserArn"`
-	AddedIAMUserToGroup bool   `json:"addedIAMUserToGroup"`
-	PartialError        bool   `json:"partialError"`
-	AccessKey           string `json:"accessKey"`
-	SecretKey           string `json:"secretKey"`
+	IAMUserName string `json:"iamUserName"`
+	IAMUserArn  string `json:"iamUserArn"`
+	AccessKey   string `json:"accessKey"`
+	SecretKey   string `json:"secretKey"`
 }
 
 // LongTermKeyRequest is used to represent the request body to create or delete LTKs
 type LongTermKeyRequest struct {
-	Account     string `json:"account"`
+	AccountDetails
```

Comment:
So I see `AccountDetails` is composed of "account" and "role", I'm confused why in our swagger docs says it's only  the "account" and "iamUserName" field are required where the account field is in the 012345678910/ALKSIAMAdmin - awscoxautolabs1 format

 https://pages.ghe.coxautoinc.com/ETS-CloudAutomation/ALKS-Documentation/#/long-term-key-rest-service/createServiceUserUsingPOST